### PR TITLE
Set up the component

### PR DESCRIPTION
**Pull request by abridger**

### CODE REVIEW

File: kwc-product-card.html [hunk 1]
```diff
@@ -0,0 +1,215 @@
...
+                        sizing="contain">
+                        </iron-image>
+        </a>
+        <div class="age">[[ageRange]]</div>
+        <div class="main">
+            <h2>[[title]]</h2>
+            <template is="dom-if" if="[[description]]">
+                <p>[[description]]</p>
+            </template>
+            <template is="dom-if" if="[[features]]">
```
Comment (murilopolese) [inline kwc-product-card.html]: In theory (as the [documentation points out](https://www.polymer-project.org/2.0/docs/api/elements/Polymer.DomIf)) the `if` should be a boolean.

File: kwc-product-card.html [hunk 2]
```diff
@@ -0,0 +1,215 @@
...
+                 * - kano-orange
+                 * - candlelight
+                 * - grassland
+                 * - sky
+                 * - flamingo
+                 */
+                color: {
+                    type: String,
+                    value: 'kano-orange',
+                    reflectToAttribute: true
```
Comment (murilopolese) [inline kwc-product-card.html:180]: Why do we want to reflect this as an attribute?

**Review by murilopolese — CHANGES_REQUESTED** [review]
Just a little thing with the `dom-if` and a question, otherwise looks good. We might come up with a better way to test/populate fixtures so we don't rely so much on comparing with literals...

File: Jenkinsfile [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+                    sh "polymer lint"
+                    sh "polymer test --local chrome"
+                    junit allowEmptyResults: true, testResults: 'wct.xml'
+                }
+            }
+        }
+
+        stage('documentation') {
+            steps {
+                script {
```
Comment (paulvarache) [inline Jenkinsfile:41]: Can update with
```groovy
script {
    if (env.BRANCH_NAME == 'master') {
        build job: 'Kano/components-doc/master', parameters: [
            text(name: 'repoUrl', value: 'https://github.com/KanoComponents/kwc-product-card'),
            text(name: 'componentName', value: 'kwc-product-card')
        ], wait: false
    }
}
```

To get the docs added to https://components.kano.me

File: kwc-product-card.html (hunk 2 above)
Comment (abridger) [reply kwc-product-card.html:180]: It gets used for styling. Just personal preference I guess, but I didn't want to have to generate classes :-)

Comment (murilopolese) [reply kwc-product-card.html:180]: Aha! Got it :)

File: kwc-product-card.html (hunk 1 above)
Comment (abridger) [reply kwc-product-card.html]: Sorry, yes, it just wouldn't let me use `!!features`. Growl. I'll update.

**Review by murilopolese — APPROVED** [review]

**Pull request merged**